lib/experimental/class-wp-rest-navigation-controller.php

```diff
@@ -0,0 +1,86 @@
+<?php
+/**
+ * REST API: Gutenberg_REST_Navigation_Controller class
+ *
+ * @package    Gutenberg
+ * @subpackage REST_API
+ */
+
+require __DIR__ . '/class-wp-navigation-gutenberg.php';
+
+
+/**
+ * Base Templates REST API Controller.
+ */
+class WP_REST_Navigation_Controller extends WP_REST_Posts_Controller {
+
+
+
+	/**
+	 * Registers the controllers routes.
+	 *
+	 * @return void
+	 */
+	public function register_routes() {
+
+		parent::register_routes();
+
+		// Lists a single nav item based on the given id or slug.
+		register_rest_route(
+			$this->namespace,
+			'/' . $this->rest_base . '/fallbacks',
```

Comment:
I would rename it to singular `fallback` or to `default` or something singular because we only respond with one item.

Reply:
`fallback` is is 👍